frontend/src/metabase/pulse/components/RecipientPicker.jsx

```diff
@@ -73,7 +76,8 @@ export default class RecipientPicker extends Component {
             }
             autoFocus={autoFocus && recipients.length === 0}
             multi
-            valueRenderer={value => value.common_name || value.email}
+            // https://user-images.githubusercontent.com/1937582/172163846-86636488-9cb7-4b8f-9609-594b42384f4a.png
+            valueRenderer={value => value.common_name}
```

Comment:
`common_name` already falls back to `email`

<img src="https://user-images.githubusercontent.com/1937582/172163846-86636488-9cb7-4b8f-9609-594b42384f4a.png" />